### PR TITLE
Add test for JSP with EL expression

### PR DESCRIPTION
**Pull request by tjwatson**
This test adds a JSP test that accesses CDI beans
using an expression.  The JSPs access a request
scope and application scope bean.

### CODE REVIEW

**Comment by tjwatson** [comment]
#build

**Comment by tjwatson** [comment]
Tests for #21326

File: dev/io.openliberty.checkpoint_fat/test-applications/JSPWithEL/src/jsp/with/el/ApplicationScopeBean.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+@ApplicationScoped
+@Named("asb")
+public class ApplicationScopeBean {
+    private static AtomicInteger constructCount = new AtomicInteger();
+
+    public ApplicationScopeBean() {
+        constructCount.incrementAndGet();
+    }
+
+    public void observeInit(@Observes @Initialized(ApplicationScoped.class) Object event) {
```
Comment (dazavala) [inline dev/io.openliberty.checkpoint_fat/test-applications/JSPWithEL/src/jsp/with/el/ApplicationScopeBean.java:31]: The test could further verify the observer behavior executes during checkpoint, when the application starts.  Looking at the test's checkpoint `message.log`, the behavior executes as intended.

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/WebProfileJSPWithELTest.java [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    @ClassRule
+    public static RepeatTests r = RepeatTests.withoutModification()
+                    .andWith(new JakartaEE9Action().forServers(SERVER_NAME).fullFATOnly())
+                    .andWith(new JakartaEE10Action().forServers(SERVER_NAME).fullFATOnly());
+
+    @Server(SERVER_NAME)
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void exportWebApp() throws Exception {
```
Comment (dazavala) [inline dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/WebProfileJSPWithELTest.java:53]: W.r.t. style the method name is a bit misleading as the method also invokes server checkpoint and restore.

**Review by dazavala — APPROVED** [review]
LGTM.  I left two comments for your consideration, but there are no necessary changes AFAIS.

**Comment by tjwatson** [comment]
#libby

File: dev/io.openliberty.checkpoint_fat/test-applications/JSPWithEL/src/jsp/with/el/ApplicationScopeBean.java (hunk 1 above)
Comment (dazavala) [reply dev/io.openliberty.checkpoint_fat/test-applications/JSPWithEL/src/jsp/with/el/ApplicationScopeBean.java:31]: See PR [25123](https://github.com/OpenLiberty/open-liberty/pull/25123).

File: dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/WebProfileJSPWithELTest.java (hunk 2 above)
Comment (dazavala) [reply dev/io.openliberty.checkpoint_fat/fat/src/io/openliberty/checkpoint/fat/WebProfileJSPWithELTest.java:53]: See PR [25123](https://github.com/OpenLiberty/open-liberty/pull/25123)

**Pull request merged**